.github/workflows/dependabot.yml

```diff
@@ -8,7 +8,7 @@ permissions:
 jobs:
   dependabot:
     name: Enable auto-merge
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
shouldn't it be 22.04 like in other jobs?

Reply:
Franky, I'd migrate them all to `latest` and forget about it.

Reply:
sometimes it might lead to unexpected failures

Reply:
Really? Any examples?

Reply:
I'm sure dependabot will just work, because both the runner and the actor are from the same vendor :)

Reply:
Btw, even if we pin the version, there is never a guarantee, since runners also get updated regularly without us noticing it... but they still mostly work 🤞

Reply:
https://github.com/actions/runner-images/issues/10788
https://github.com/actions/runner-images/issues/2234
https://exercism.org/docs/building/github/gha-best-practices#h-pin-test-runners-to-version
I'm sure that you are right about this case and that action will keep working, but behavior might change when new ubuntu will come with new node version and this action will still be using old node version.

Reply:
Sorry, the examples above are irrelevant IMO. Again, the runner and the bot are maintained by the same company.

On the other hand this problem is so minor, that I don't see why we should sweat over it more than that, so I concur.